File: src/main/java/kookmin/software/capstone2023/timebank/domain/repository/BankBranchJpaRepository.java
package kookmin.software.capstone2023.timebank.domain.repository;

import kookmin.software.capstone2023.timebank.domain.model.BankBranch;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface BankBranchJpaRepository extends JpaRepository<BankBranch, Long> {
}

File: src/main/java/kookmin/software/capstone2023/timebank/domain/repository/BankBranchJpaRepository.kt
package kookmin.software.capstone2023.timebank.domain.repository

import kookmin.software.capstone2023.timebank.domain.model.BankBranch
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface BankBranchJpaRepository : JpaRepository<BankBranch, Long>
